libs/map/src/lib/components/layer/vl-map-layer.wctest.ts

```diff
@@ -73,7 +73,7 @@ const wmtsLayerFixture = async () =>
     fixture(html`
         <vl-map>
             <vl-map-wmts-layer
-                data-vl-url="https://tile.informatievlaanderen.be/ws/raadpleegdiensten/wmts"
+                data-vl-url="https://geo.api.vlaanderen.be/ws/raadpleegdiensten/wmts"
```

Comment:
ter info; ik denk dat die url in de wc test eigenlijk geen invloed heeft omdat die niets met de laag in kwestie zelf aftoetsen)

Misschien is het dan beter hier een fictieve url te plaatsen `http://dummy/wmts` of dergelijks.. zoals in de andere testen.

Laat ik aan UIG team om te beslissen

Reply:
Zolang de test blijft werken bij het meegeven van een dummy url mag je het gerust aanpassen.

Reply:
Aangepast, lokaal waren wc testen ok. Maar toch even Bamboo afwachten to be sure

Reply:
Done